classes/Context.php

```diff
@@ -347,6 +347,12 @@ public function getTranslator()
             return $this->translator;
         }
 
+        global $kernel;
+        if ($kernel instanceof Symfony\Component\HttpKernel\HttpKernelInterface) {
```

Comment:
That's true, and because I copy pasted this code, we have other occurrences to modify!